Include linux/if_tun.h after netinet/if_tun.h to avoid redefinition of struct ethhdr

diff --git a/src/if-linux.c b/src/if-linux.c
index fd47278..cca4e36 100644
--- a/src/if-linux.c
+++ b/src/if-linux.c
@@ -37,7 +37,6 @@
 #include <linux/if_addr.h>
 #include <linux/if_link.h>
 #include <linux/if_packet.h>
-#include <linux/if_tun.h>
 #include <linux/if_vlan.h>
 #include <linux/filter.h>
 #include <linux/netlink.h>
@@ -51,6 +50,8 @@
 #include <netinet/in.h>
 #include <net/route.h>
 
+#include <linux/if_tun.h>
+
 #include <errno.h>
 #include <fcntl.h>
 #include <ctype.h>

